schemas/bedrock-vc-issuer.js

```diff
@@ -68,8 +68,17 @@ export const issueCredentialBody = {
   additionalProperties: false,
   properties: {
     options: {
-      // FIXME: make restricted
-      type: 'object'
+      type: 'object',
+      additionalProperties: false,
+      properties: {
+        mandatoryPointers: {
+          type: 'array',
+          minItems: 1,
```

Comment:
I guess you could technically have a case where you want this to be `0` at this time ... to clear all mandatory fields. We may up it later, but let's leave it at `0` for now.

```suggestion
          minItems: 0,
```